### ISSUE TITLE
MergeOption in Entity Framework

### ISSUE THREAD

**Issue by tobimax** [riteshrao/ncommon#27]
Hi,

Does NCommon support MergeOption.NoTracking etc for the EF Repositories?

Regards

Toby.

**Comment by riteshrao** [comment]
Hi Toby,

Sorry for the late response, last couple of days have been a little hectic.
NCommon's EF repository does not handle setting the merge option of queries.
In what scenarios are you looking to set the MergeOption?

Thanks,
Ritesh

**Comment by tobimax** [comment]
No, worries, thanks for coming back to me. In EF 4 you can use for example MergeOption.NoTracking to effectivley pre- detach your entities, improves performance slighlty. I guessed it didn't support it as the base class & inetrface doesn't have it in there so I actually made a change to the code to suit my senario.

regards

Toby
